docs/tcp_replication.md

```diff
@@ -174,9 +174,8 @@ client (C):
 
 #### POSITION (S)
 
-   The position of the stream has been updated. Sent to the client
-    after all missing updates for a stream have been sent to the client
-    and they're now up to date.
+   On receipt of a POSITION command clients should check if they have missed any
```

Comment:
am I right in thinking it is only sent on connect? might be worth saying that? or otherwise saying when it is sent?

Reply:
Generally happens in response to a `REPLICATE` command, but in a redis world they'll be flying around whenever anyone connects and sends a `REPLICATE`. Will update comment.